docs/sidebars.ts

```diff
@@ -44,6 +52,45 @@ const sidebars: SidebarsConfig = {
       ],
     },
     "ui",
+    {
+      type: "category",
+      label: "API Reference",
+      // collapsible: true,
+      // collapsed: true,
```

Comment:
```suggestion
      collapsible: true,
      collapsed: true,
```

consistency with the other entries + seems like this is the default anyway?